Ccde-guide.html

```
@@ -0,0 +1,1499 @@
+                    </div>
+                </div>
+                <div class="sheet-body">
+                    <p>
+                        CCDE認定を取得するまでの流れは、大きく分けて「筆記試験」と「実技試験」の2段階です。まずは全体像を1枚のフローチャートで確認しましょう。
+                    </p>
+
+                    <div class="diagram-card">
+                        <pre class="mermaid">
+flowchart TD
+    Pre["実務経験 5〜7年を推奨<br/>（正式な出願資格は無し）"]
+    Pre --> Step1["STEP 1<br/>CCDE筆記試験 400-007 v3.1<br/>2時間・90〜110問"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)